docs/modules/datasources/pages/performance_datasource.adoc

```diff
@@ -88,6 +88,11 @@ The results will appear as a drop-down list at the top of the dashboard.
 You can then select from the list to change the data the panel displays.
 See xref:panel_configuration:dynamic-dashboard.adoc#pc-template-filters[Using Template Variables to Create Dynamic Dashboards] for an example.
 
+=== nodeLocation()
```

Comment:
IMO, this should be plural. It does not give the location of a node but all relevant location**s**.

Reply:
Thinking some more, this should be `locations` only. By wording, there is no "Node Location h but only a "Location" where Nodes (and other stuff) are located in.

Reply:
Applied change